src/app/pages/LandingPage/LandingPage.module.css

```diff
@@ -0,0 +1,14 @@
+.container {
+  display: grid;
+  grid-template-rows: 60% 1fr 1fr 25%;
+  justify-items: center;
+  align-items: end;
+  row-gap: 2em;
+  margin: 6em 2.5em 0;
+  max-width: 30em;
+}
+
+.logo {
+  width: 90%;
```

Comment:
This could look weird for big screens. Did you test it on different resolutions? A max width could be useful.

Reply:
Do I need a max width here if the App has a max width?

Reply:
No :)